photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -415,9 +421,12 @@ object GameTrainingDriver extends GameDriver {
         case None => None
       }
     }
+    val intercepts: Map[FeatureShardId, Option[Int]] = featureIndexMapLoaders.mapValues{
+        _.indexMapForDriver()
+        .get(Constants.INTERCEPT_KEY)}
```

Comment:
we cannot do it in calculateStatistics since the normalization context uses the functions too. but we cannot exclude the intercept for normalization because some feature projection logic is affected there when the summary and feature does not match. I have encountered a bug there. So I think our current solution is to do it in the calculateAndSaveFeatureShardStats.

Reply:
Let me double check the code, I think I understand.

Reply:
I'm working on a separate commit that should go in before this one - it will clear up the trouble with the intercept for now, and let us deal with it later.